react 2/my-blog/my-blog/app/post/[post]/page.js

```diff
@@ -0,0 +1,9 @@
+export default function Post({params}) {
+    console.log(params.post);
+    let title = params.post.split("-");
+    title =title.join (" ").toUpperCase();
+    return (
+      <h1>{title}</h1>
+    );
+  }
```

Comment:
I can see that this thing didn't work.

Next was trying to give you information what might be wrong with it: `params` is asynchronous API. Which means that `Post` should be `async` and if you want to get a parameter from it, code should look like this:
```
const data = await params;
const title = data.post.split('=');
...
```

<img width="554" alt="image" src="https://github.com/user-attachments/assets/16037fa9-b460-47c9-b987-ddedd33b4993">

Reply:
i made changes